# phenoman
Phenotypic data exploration, selection, management and quality control for association studies of rare and common variants
 
* Published on Bioinformatics Journal [link] (https://www.ncbi.nlm.nih.gov/pubmed/24336645)

## Acknowledgment
Development of PhenoMan was supported by the NHLBI Exome Sequencing Project, the Minority Health GRID project, and the Centers for Mendelian Genomics.

## Installation and Use Guide
* Refer to [Documentation] ()
	* Download [source code] () and [example data] () 

## Introduction
Recently, the next generation sequencing and other high-throughput technology advances to promote great interest in detecting associations between complex phenotypic traits and genetic variants. Phenotype quality control procedure is crucial and could be a complicated issue that can largely impact association analysis results. Although various decisions are likely to be made on different traits, there is lacking a simple, effective and uniformed way to perform and record phenotype data cleaning steps. Arbitrariness and ambiguity often arise in managing phenotype data quality. This problem is normally neglected, which could cause biased meta-analysis or comparison of association results between studies on same traits. PhenoMan is an interactive program that integrates data exploration, management and quality control using a unified platform. It is featured by dissecting ambiguous data cleaning steps into a series of simple commands with dynamic arguments, which are clearly recording the entire data cleaning procedure. PhenoMan provides approaches in efficient exploration and management of phenotype data. We should perform them before the association analysis so as to ensure effective estimates from association results and consistent comparisons between different projects. PhenoMan can be used on new and existing rich sets of phenotypic data for association analysis of both quantitative and qualitative traits.




